### PR TITLE
[DEVOPS-382] staging infra master

### PR DESCRIPTION
**Pull request by deepfire**
This is a sister PR to https://github.com/input-output-hk/iohk-ops/pull/191

### CODE REVIEW

File: iohk/iohk-ops.hs [hunk 1]
```diff
@@ -165,8 +164,8 @@ centralCommandParser =
      --                           (Nixops
      --                            <$> (NixopsCmd <$> argText "CMD" "Nixops command to invoke")
      --                            <*> ???)) -- should we switch to optparse-applicative?
-     ("create",                 "Create the whole cluster",                                         pure Create)
-   , ("modify",                 "Update cluster state with the nix expression changes",             pure Modify)
+     ("modify",                 "Update cluster state with the nix expression changes",             pure Modify)
+   , ("create",                 "Same as modify",                                                   pure Modify)
```
Comment (jmitchell) [inline iohk/iohk-ops.hs]: Should we just remove `create`? I see the internal change happened at 605a4e4.

File: deployments/infrastructure.nix [hunk 2]
```diff
@@ -43,44 +42,14 @@ in {
...
-          "/etc/"
-        ];
-      };
-    };
-
     networking.firewall.allowedTCPPortRanges = [
       { from = 24962; to = 25062; }
     ];
   };
+  resources = {
```
Comment (domenkozar) [inline deployments/infrastructure.nix]: This should be in `deployments/infrastructure-target-aws.nix`

File: iohk/iohk-ops.hs [hunk 3]
```diff
@@ -313,9 +311,6 @@ runNew o@Options{..} New{..} args = do
   echo $ "-- " <> (unsafeTextToLine $ configFilename) <> " is:"
   cmd o "cat" [configFilename]
```
Comment (domenkozar) [inline iohk/iohk-ops.hs]: There should be a call to create here. The idea is to make sure it's either created or modified. We also want to bring create back to backwards compatibitily

File: iohk/iohk-ops.hs (hunk 1 above)
Comment (deepfire) [reply iohk/iohk-ops.hs]: Right, missed that..

Comment (deepfire) [reply iohk/iohk-ops.hs]: Ah, I didn't miss that, actually, it's an alias to modify, precisely for backward compat -- already forgot this..

File: iohk/iohk-ops.hs (hunk 3 above)
Comment (deepfire) [reply iohk/iohk-ops.hs]: @domenkozar, `io deploy` runs `io modify` (which should be probably renamed into `update`), which runs either `nixops modify` or `nixops create`, as appropriate.

Also, `io create` is still there and it runs `io modify`.

File: deployments/infrastructure.nix (hunk 2 above)
Comment (deepfire) [reply deployments/infrastructure.nix]: Done..

**Pull request merged**